lib/src/components/NxTag/__tests__/NxTag.test.tsx

```diff
@@ -4,124 +4,125 @@
  * the terms of the Eclipse Public License 2.0 which accompanies this
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
-import React from 'react';
-import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
-import 'jest-enzyme';
-import { shallow, mount } from 'enzyme';
-import { faPlusCircle, faTimesCircle } from '@fortawesome/free-solid-svg-icons';
+import React, { RefAttributes } from 'react';
 
-import NxTag, { NxSelectableTag } from '../NxTag';
-import NxFontAwesomeIcon from '../../NxFontAwesomeIcon/NxFontAwesomeIcon';
+import NxTag, { NxSelectableTag, PublicProps, SelectableProps } from '../NxTag';
+
+import { rtlRender, rtlRenderElement, userEvent } from '../../../__testutils__/rtlUtils';
 
 describe('NxTag', function() {
-  const getShallowComponent = enzymeUtils.getShallowComponent(NxTag, { children: 'basic tag' });
+  type PropsWithRef = PublicProps & RefAttributes<HTMLLabelElement>;
+  const renderEl = rtlRenderElement<PropsWithRef>(NxTag, { children: 'basic tag '});
 
   it('renders NxTag with the `nx-tag` class', function() {
-    expect(getShallowComponent().children().find('.nx-tag')).toExist();
+    const el = renderEl()!;
+    expect(el.tagName).toBe('LABEL');
+    expect(el).toHaveClass('nx-tag');
   });
```

Comment:
IMO both the classname and the tagName are impl details

Reply:
> IMO both the classname and the tagName are impl details

Removed here: 607f83a034413c21cc13758c8e42ec8a7c0ba678